### PR TITLE
Fixed an issue where lock layout menu was not in sync with preferences. #5758

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/browser/templates/browser/js/utils.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
   {% endif %}label: "{{ item.label }}", applies: ["{{ key.lower() }}"],
   priority: {{ item.priority }},
   enable: "{{ item.enable }}",
-  {% if item.checked is defined %}checked: {% if item.checked %}true{% else %}false{% endif %},
+  {% if item.checked is defined %}checked: {% if item.name == 'mnu_lock_'+current_ui_lock %}true{% else %}false{% endif %},
```
Comment (adityatoshniwal) [inline web/pgadmin/browser/templates/browser/js/utils.js:23]: There should be one more condition for `{% if item.checked %}`

File: web/pgadmin/preferences/static/js/components/PreferencesComponent.jsx [hunk 2]
```diff
@@ -574,6 +574,23 @@ export default function PreferencesComponent({ ...props }) {
             gettext('Later')
           );
         }
+        /*
```
Comment (adityatoshniwal) [inline web/pgadmin/preferences/static/js/components/PreferencesComponent.jsx:577]: Change the comment:
`Sync the lock layout menu with preferences`

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
Otherwise, it looks good to me.

File: web/pgadmin/browser/templates/browser/js/utils.js (hunk 1 above)
Comment (pravesh-sharma) [reply web/pgadmin/browser/templates/browser/js/utils.js:23]: iten.checked will either be true or false and  was earlier used  to set checked property to true or false, I think adding item.checked will not have any effect. please let me know if I am missing any point here.

**Pull request merged**